src/state/download/actions.test.js

```diff
@@ -39,7 +39,25 @@ describe('fetchDataSet', () => {
 
     await store.dispatch(fetchDataSet());
 
-    expect(store.getActions().map((x)=> x.type)).toEqual(["DOWNLOAD_DATASET_FETCH", "DOWNLOAD_CLEAR", "DOWNLOAD_CLEAR_SUCCESS", REPORT_ERROR]);
+    expect(store.getActions().map((x)=> x.type)).toEqual(["DOWNLOAD_DATASET_FETCH", "DOWNLOAD_CLEAR", REPORT_ERROR]);
```

Comment:
Trivial question: It seems that most of the front-end code adds a space before and after `=>`. Can you also add a space before `=>` here?

Reply:
Sure! I'm don't know why this wasn't picked by prettify